### ISSUE TITLE
Windows 10 Spyder Shortcut Launches Spyder with broken kernel

### ISSUE THREAD

**Issue by PhilipYip1988** [ContinuumIO/anaconda-issues#11678]
### Actual Behavior

Spyder 4.1.0 launches from Windows shortcut. Kernal fails to load properly.

### Expected Behavior

Spyder 4.1.0 launches with Kernal. Works when launched from Anaconda navigator.

### Steps to Reproduce

Install Anaconda 2020-02
Open up Anaconda Power Shell Prompt
Type in conda update anaconda
Launch Spyder from shortcut on Start Menu

##### `conda info`

<details>

```
     active environment : base
    active env location : C:\Users\pyip\Anaconda3
            shell level : 1
       user config file : C:\Users\pyip\.condarc
 populated config files : C:\Users\pyip\.condarc
          conda version : 4.8.3
    conda-build version : 3.18.11
         python version : 3.7.6.final.0
       virtual packages :
       base environment : C:\Users\pyip\Anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\pyip\Anaconda3\pkgs
                          C:\Users\pyip\.conda\pkgs
                          C:\Users\pyip\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\pyip\Anaconda3\envs
                          C:\Users\pyip\.conda\envs
                          C:\Users\pyip\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.8.3 requests/2.23.0 CPython/3.7.6 Windows/10 Windows/10.0.19041
          administrator : True
             netrc file : None
           offline mode : False


```

</details>

##### `conda list --show-channel-urls`

<details>

```
PASTE OUTPUT HERE:

```

</details>
![vlcsnap-2020-03-19-12h46m17s164](https://user-images.githubusercontent.com/56493395/77069084-ada33800-69df-11ea-8be8-60117c7baa65.png)

**Comment by ccordoba12** [comment]
This problem will be fixed in Spyder 4.1.1, to be released today or tomorrow.

**Comment by PhilipYip1988** [comment]
Just installed Spyder 4.1.1 on two Windows 10 machines it seems to have fixed the issue. Thanks.
